src/frontend/efiling-frontend/src/components/page/home/Home.js

```diff
@@ -43,7 +43,8 @@ const checkCSOAccountStatus = (
         const csoAccountExists = userDetails.accounts.some(
           element => element.type === "CSO"
         );
-        if (csoAccountExists) setCsoAccountExists(true);
+        if (csoAccountExists)
```

Comment:
do we care if it doesn't?

Reply:
if it does not exist, then the current state will handle it for now. later when we know our error handling approach, we will do some custom stuff :)